app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> str:
+
+    parts = command.split()
+
+    if len(parts) < 3:
+        return "Invalid command. Please provide source and destination paths."
```

Comment:
u don't need to return a string here
just print the information and return nothing
```return```